src/Sentry.Unity/SentryInitialization.cs

```diff
@@ -1,197 +1,45 @@
-using System;
-using System.Collections.Generic;
 using UnityEngine;
-using UnityEngine.SceneManagement;
-using CompressionLevel = System.IO.Compression.CompressionLevel;
 
 namespace Sentry.Unity
 {
-    public interface IEventCapture
-    {
-        SentryId Capture(SentryEvent sentryEvent);
-    }
-
-    internal class EventCapture : IEventCapture
-    {
-        public SentryId Capture(SentryEvent sentryEvent)
-            => SentrySdk.CaptureEvent(sentryEvent);
-    }
-
     // https://94677106febe46b88b9b9ae5efd18a00@o447951.ingest.sentry.io/5439417
     public static class SentryInitialization
     {
-        // TODO: Stuff that should be passed with https://github.com/getsentry/sentry-unity/issues/66 implementation
-        internal static IEventCapture EventCapture = new EventCapture();
-        internal static ErrorTimeDebounce ErrorTimeDebounce = new(TimeSpan.FromSeconds(1));
-        internal static LogTimeDebounce LogTimeDebounce = new(TimeSpan.FromSeconds(1));
-        internal static WarningTimeDebounce WarningTimeDebounce = new(TimeSpan.FromSeconds(1));
-
-        internal static bool IsInit { get; private set; }
-
         [RuntimeInitializeOnLoadMethod(RuntimeInitializeLoadType.BeforeSceneLoad)]
         public static void Init()
         {
             var options = UnitySentryOptions.LoadFromUnity();
 
-            if (!options.Enabled)
-            {
-                options.Logger?.Log(SentryLevel.Debug, "Disabled In Options.");
-                return;
-            }
-
-            if (!options.CaptureInEditor && Application.isEditor)
+            if (!options.DisableProgrammaticInitialization)
             {
-                options.Logger?.Log(SentryLevel.Info, "Disabled while in the Editor.");
+                /*
+                 * Call Unity's `Debug.Log()` directly via Sentry `UnityLogger`
+                 * We want to display the message in spite of how SentryOptions.json is configured
+                 */
+                new UnityLogger(SentryLevel.Info).Log(SentryLevel.Info, "Programmatic access enabled. Configure Sentry manually.");

```

Comment:
Why not call `options.DiagnosticLogger` here?

And if really not, why not just `Debug.Log` then?

Reply:
There are a couple of reasons behind this approach. My reasoning is the following

# First question

In order to use `SentryOptions.DiagnosticLogger` it should be specifically configured in `SentryOptions`. For example, it should have `SentryOptions.Debug` set to `true` **and** it should have some object assigned to `SentryOptions.DiagnosticLogger` already (ie. `UnityLogger`) **and** some other Unity specific conditions like `!DebugOnlyInEditor || Application.isEditor`. 
 
If user disables Sentry Window we should display the message **in spite of** what was set in `SentryOptions`. I think we should not expect a user to properly configure `SentryWindow` to enable `UnityLogger`, then disable the whole thing so he can see a message.

In order to do it in `SentryOptions` context and not rely on `SentryWindow` settings:
```csharp
options.Debug = true;
options.DiagnosticLogger = new UnityLogger(SentryLevel.Info);
options.DiagnosticLogger.Log(SentryLevel.Info, "Programmatic access enabled. Configure Sentry manually.");
```
but this is not needed as we can just call `UnityLogger` directly.

# Second question

`UnityLogger` has specific format that it outputs into Unity. Yes, we can call it directly via Unity, but we probably want to have Sentry specific formatting applied so the user can differentiate that this message came from Sentry specifically.

Reply:
@bruno-garcia is the current explanation fine for you? Can I resolve it?

Reply:
Yeah that makes.